### PR TITLE
docs: Added an "Override a React context" guide

### PR DESCRIPTION
**Pull request by patricklafrance**
- Added a new "Override a React context" guide
- CHat GTPed the "Troubleshooting" page

### CODE REVIEW

File: docs/guides/federated-tabs.md [hunk 1]
```diff
@@ -279,6 +279,35 @@ export default function FederatedTabsLayout() {
...
+
+Similarly to how the display order of regular navigation items can be configured, a federated tab position can be affected with the [priority](http://localhost:5000/wl-squide/reference/runtime/runtime-class/#sort-registered-navigation-items) property.
+
+To force `Tab 3` to be positioned first: 
+
+```tsx !#17 local-module/src/register.tsx
+import type { ModuleRegisterFunction, Runtime } from "@squide/react-router";
+import { Tab3 } from "./Tab3.tsx";
+
+export const register: ModuleRegisterFunction<Runtime> = (runtime: Runtime) => {
```
Comment (alexasselin008) [inline docs/guides/federated-tabs.md:292]:
```suggestion
export const register: ModuleRegisterFunction<Runtime> = runtime => {
```

File: docs/guides/overriding-a-react-context.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+```tsx !#16-21 host/src/App.tsx
+import { useAppRouter } from "@sample/shell";
+import { BackgroundColorContext } from "@sample/shared";
+import { useAreModulesReady } from "@squide/webpack-module-federation";
+import { RouterProvider } from "react-router-dom";
+
+export function App() {
+    const isReady = useAreModulesReady();
+
```
Comment (alexasselin008) [inline docs/guides/overriding-a-react-context.md:20]: extra line break for no reason?

File: docs/guides/overriding-a-react-context.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+```tsx !#16-21 host/src/App.tsx
+import { useAppRouter } from "@sample/shell";
+import { ThemeContext } from "@hopper/components";
+import { useAreModulesReady } from "@squide/webpack-module-federation";
+import { RouterProvider } from "react-router-dom";
+
+export function App() {
+    const isReady = useAreModulesReady();
+
```
Comment (alexasselin008) [inline docs/guides/overriding-a-react-context.md:131]: extra line break here also

**Review by alexasselin008 — APPROVED** [review]

File: docs/guides/overriding-a-react-context.md (hunk 2 above)
Comment (patricklafrance) [reply docs/guides/overriding-a-react-context.md:20]: Oh I almost always had one between my hooks, all the samples of Squide has this line break!

**Review by alexasselin008 — APPROVED** [review]

**Pull request merged**